File: Programmers/C++/Level_1/Lv1_이상한_문자_만들기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string s) {
    string answer = "";
    int idx = 0;
    
    for (int i=0; i<s.size(); i++){
        if (s[i] == ' ') {
            idx = 0;
            answer += s[i];
            continue;
        }
        
        if (idx % 2 == 0)
            answer += toupper(s[i]);
        else
            answer += tolower(s[i]);
        idx++;
    }
    
    return answer;
}

File: Programmers/Python/Level_1/Lv1_이상한_문자_만들기.py
def solution(s):
    s_split = s.split(" ")
    
    for i in range(len(s_split)):
        s_list = list(s_split[i])
        
        for j in range(len(s_list)):
            if j%2 == 0:
                s_list[j] = s_list[j].upper()
            else:
                s_list[j] = s_list[j].lower()
        s_split[i] = "".join(s_list)
    answer = " ".join(s_split)
    return answer